### DISCUSSION TITLE
是否需要修改六角龙机器人在发起功能投票discussion的正文内容

### DISCUSSION THREAD

**Discussion by Deep-Dark-Forest** [Meloong-Git/PCL#3516] [讨论]
一堆人在功能投票里面说“支持”，这种无意义的评论对我们造成了严重的打扰
要不，功能投票讨论的末尾可以改成“如果你支持此功能，你可以点击左下角的`↑`。如果你不想表达其他观点，请勿发表任何评论，这会对很多人产生不必要的打扰。”

**Comment by iPodtouch6557** [comment]
霞客怎样都是霞客，没啥必要（故意打错的

**Reply by Deep-Dark-Forest** [reply]
~~还是有一点用的，毕竟对于有些新来的人他们都会好好看每一行文字~~（算了，还是幻想时间）

**Comment by Deep-Dark-Forest** [comment]
没人理我啊啊啊

**Reply by WorldHim** [reply]
话说为什么 `Hex Dragon` 要翻译成 `六角龙`

**Reply by allMagicNB** [reply]
> 话说为什么 `Hex Dragon` 要翻译成 `六角龙`

![Screenshot_20240315_131422.jpg](https://github.com/Hex-Dragon/PCL2/assets/116283787/b029e672-a123-4a82-912e-90bd21cfa397)

微软特色

**Reply by wuliaodexiaoluo** [reply]
因为 Hex = 六角
又因为 Dragon = 龙
所以 Hex + Dragon = 六角 + 龙 = 六角龙

**Reply by WorldHim** [reply]
这显然不够信达雅？
#3526

**Reply by hejiehao** [reply]
Hex 好像也有 16 进制的意思

**Reply by iPodtouch6557** [reply]
Hexagon是六边形

**Comment by wuliaodexiaoluo** [comment]
发了个 Issue 看看龙猫怎么说

**Comment by Light-Beacon** [comment]
其实感觉上边写的东西就没人读（（
~~btw 六角龙其实你可以翻译成 *我超勇的*（我和六角龙无关，仅个人观点）~~
